metabase/metabase
# Migrate filter field values to RTK

Migrate field values endpoints used in question filters and relevant rescan/discard endpoints in Admin settings to RTK.

How to verify:
- New -> Question -> Products
- Add filter -> Category. Make sure there is a `values` endpoint call.
- Close the picker and open it again. There should no additional call.
- Go to Admin settings -> Table Metadata -> Products -> Category -> Rescan field values -> Exit admin
- Go back to the question and try to add a filter for Category. There should be a `values` request.
